.github/workflows/publish.yml

```diff
@@ -92,3 +92,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_CONFIG_PROVENANCE: true
+
+      - uses: codecov/codecov-action@v5
```

Comment:
after we publish everything, we can try to upload a coverage report. doing this after because by this point we should have 

A) run tests, and builds 
B) I don't want uploading coverage to affect the publish or versioning, etc